src/deploy.ps1

```diff
@@ -82,5 +91,31 @@ function PublishToBlobStorage {
 	Set-AzureStorageBlobContent -File $targetDllPath -Container $BlobStorageContainer -Blob $blobName -Context $blobContext -Force
 }
 
+
+function UploadAssembliesToAdls() {
+	# Check to see whether we need to upload to blob
+	if([string]::IsNullOrEmpty($AzureDataLakeStoreName)) {
+        Write-Host "No data lake information specified. Not uploading to storage."
+		return
+	}
+
+	Login-AzureRmAccount
+	Set-AzureRMContext -SubscriptionId $SubscriptionId 
```

Comment:
The only problem here is it's launching interactive login. We can skip login altogether, assuming that PowerShell is already in session and subscription is selected.